### PR TITLE
update missing Chinese translation for v2.6

### PR DESCRIPTION
**Pull request by smallteeths**
@JacieChao please help me to reviews the docs when you have free time, thanks alog.

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/version-2.6/how-to-guides/new-user-guides/kubernetes-resources-setup/kubernetes-resources-setup.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## 服务发现
+
+使用负载均衡器和/或 Ingress 将集群公开给外部请求后，你只能通过 IP 地址访问集群。要创建可解析的主机名，你必须创建服务记录，该记录将 IP 地址、外部主机名、DNS 记录别名、工作负载或标记的 pod 映射到特定主机名。
+
+有关详细信息，请参阅[服务发现](create-services.md)。
+
+## 流水线
+
+在将您的项目[配置到版本控制提供商](../../../reference-guides/pipelines/pipelines.md#1-configure-version-control-providers)之后，您可以添加仓库并开始为每个仓库配置流水线。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.6/how-to-guides/new-user-guides/kubernetes-resources-setup/kubernetes-resources-setup.md:56]: ../../../reference-guides/pipelines/pipelines.md#1-configure-version-control-providers => ../../../reference-guides/pipelines/pipelines.md#1-配置版本控制提供程序

Comment (smallteeths) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.6/how-to-guides/new-user-guides/kubernetes-resources-setup/kubernetes-resources-setup.md]: done

**Review by JacieChao — COMMENTED** [review]
Thanks. LGTM

**Review by sunilarjun — APPROVED** [review]
Thank you!

**Pull request merged**